addons/hr_recruitment/models/hr_recruitment.py

```diff
@@ -434,6 +434,40 @@ def create_employee_from_applicant(self):
         if employee:
             dict_act_window['res_id'] = employee.id
         dict_act_window['view_mode'] = 'form,tree'
+        if len(self) == 1: #only if we create one employee: print rainbow
+            #case of fast recruitment (priority 3)
+            if  (datetime.strptime(self.create_date, '%Y-%m-%d %H:%M:%S') + timedelta(days=7)).date() > date.today():
```

Comment:
We have methods in Odoo that deal with the server format, like fields.Date.fromstring . Using it allows to hide details like format.